usaspending_api/api_docs/markdown/data_dictionary.md

```diff
@@ -0,0 +1,750 @@
+<ul class="nav nav-stacked" id="sidebar">
+  <li><a href="#lexicon">Lexicon</a></li>
+  <li><a href="#endpoint-index">Endpoint Index</a></li>
+  <li><a href="#endpoint-details">Endpoint Details</a></li>
+  <li>
+    <ul>
+      <li><a href="#award">Award Endpoints</a></li>
+      <li><a href="#transaction">Transaction Endpoints</a></li>
+      <li><a href="#accounts">Accounts Endpoints</a></li>
+      <li><a href="#references">Reference Endpoints</a></li>
+      <li><a href="#submission">Submission Endpoint</a></li>
+    </ul>
+  </li>
+</ul>
+[//]: # (Begin Content)
+
+## Lexicon <a name="lexicon"></a>
+
+In this section you will find definitions for common terms used in the API. For detailed explanations of specific fields, please see that endpoint's entry in the Endpoint Details section.
+
+| Term | Explanation |
```

Comment:
Once we have a better place for it to live, I'm fine with either. I'm not sue if it will live here or not, but as it isn't on the main page now I've included it here for usability's sake.